jboss-kie-process-migration/tests/bats/jboss-kie-process-migration.bats

```diff
@@ -2,136 +2,95 @@
 
 export JBOSS_HOME=$BATS_TMPDIR/jboss_home
 export LAUNCH_DIR=$JBOSS_HOME/bin/launch
-export CONFIG_DIR=$JBOSS_HOME/configuration
-mkdir -p $LAUNCH_DIR
+export CONFIG_DIR=$JBOSS_HOME/quarkus-app/config
+mkdir -p $LAUNCH_DIR ${CONFIG_DIR} $JBOSS_HOME/quarkus-app/lib/deployment ${JBOSS_HOME}/extra-classpath/
 
 cp $BATS_TEST_DIRNAME/../../../tests/bats/common/launch-common.sh $JBOSS_HOME/bin/launch
 cp $BATS_TEST_DIRNAME/../../../tests/bats/common/logging.bash $JBOSS_HOME/bin/launch/logging.sh
+cp $BATS_TEST_DIRNAME/../../../jboss-kie-process-migration/added/launch/jboss-kie-pim-common.sh $JBOSS_HOME/bin/launch/jboss-kie-pim-common.sh
 cp $BATS_TEST_DIRNAME/../../../jboss-kie-common/added/launch/jboss-kie-common.sh $JBOSS_HOME/bin/launch/jboss-kie-common.sh
 
 #imports
 source $BATS_TEST_DIRNAME/../../added/launch/jboss-kie-process-migration.sh
 
 setup() {
-  cp -r $BATS_TEST_DIRNAME/../../added/configuration $JBOSS_HOME/configuration
+    cp -rv $BATS_TEST_DIRNAME/../../added/configuration ${CONFIG_DIR}
 }
 
 teardown() {
     rm -rf $JBOSS_HOME
 }
 
 @test "check if extra classpath is set correctly" {
-  local expected
-
-  JBOSS_KIE_EXTRA_CLASSPATH="mysql-driver.jar"
-  expected="-Dthorntail.classpath=mysql-driver.jar"
-  configure_extra_classpath >&2
-  echo "JBOSS_KIE_EXTRA_CLASSPATH is ${JBOSS_KIE_EXTRA_CLASSPATH}" >&2
-  echo "Expected is ${expected}" >&2
-  [[ $JBOSS_KIE_EXTRA_CLASSPATH == "${expected}" ]]
+    touch ${JBOSS_HOME}/extra-classpath/mysql-driver.jar
+    JBOSS_KIE_EXTRA_CLASSPATH="${JBOSS_HOME}/extra-classpath/mysql-driver.jar"
```

Comment:
Why is not included the PostgreSQL driver and is used only mysql ?

Reply:
it is just a fictitious name, doesn't really matter.